src/phpDocumentor/FlyFinder/SpecificationFactory.php

```diff
@@ -0,0 +1,91 @@
+<?php
+/**
+ * This file is part of phpDocumentor.
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ *
+ * @copyright 2010-2015 Mike van Riel<mike@phpdoc.org>
+ * @license   http://www.opensource.org/licenses/mit-license.php MIT
+ * @link      http://phpdoc.org
+ */
+
+namespace phpDocumentor\FlyFinder;
```

Comment:
Perhaps we should move this to `phpDocumentor\Infrastructure\FlyFinder`; this is not part of our Domain Model